tdrs-backend/tdpservice/search_indexes/tests.py

```diff
@@ -0,0 +1,5 @@
+"""Tests for elasticsearch model mapping."""
+
+# from django.test import TestCase
+
+# Create your tests here.
```

Comment:
i'd _like_ to write some tests, but I'm not sure what would be useful here. Also, neither `django-elasticsearch-dsl` nor `django-elasticsearch-dsl-drf` provides any documented testing capabilities. Open to discussing it further, but we probably just want to write tests around our document parsing logic when that is added.

Reply:
There is a pytest package for elasticsearch: https://pypi.org/project/pytest-elasticsearch/. We should be able to mock response a search result, etc

Reply:
good find, thanks @raftmsohani!

Reply:
I've introduced some tests - I chose not to use a mocking library and instead relied on elastic running in docker to perform integration tests. We do this in other places (clam-av scans, etc) so there's some precedent, and it seems to be the preferred method from the articles I read. It basically didn't need any extra configuration.